src/appmixer/ringring/core/MessageApiSmsRequest_Post/MessageApiSmsRequest_Post.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const lib = require('../../lib');
+
+module.exports = {
+
+    receive: async function(context) {
+
+        const { data } = await this.httpRequest(context);
+
+        return context.sendJson(data, 'out');
+    },
+
+    httpRequest: async function(context) {
+
+        // eslint-disable-next-line no-unused-vars
+        const input = context.messages.in.content;
+
+        let url = lib.getBaseUrl(context) + '/sms/v1/message';
+
+        const headers = {};
+
+        const inputMapping = {
+            'apiKey': input['apiKey'],
```

Comment:
I noticed that the `apikey` is present in this file. Could you please move it to the `auth.js` file?

Reply:
Please see https://github.com/clientIO/appmixer-components/issues/1661#issuecomment-1914767748 where I mentioned this. The problem is that OpenAPI spec [doesn't support](https://swagger.io/specification/#security-scheme-object) `apiKey` in `body`.

Reply:
Before releasing the generator, we need to improve it or change the code.

Reply:
Based on today's discussion we don't need to do any changes to the generator. When specified like this with `apiKey` in `body` we simply won't use Authentication section for the component.